src/components/VmDetails/cards/UtilizationCard/NetworkingCharts.js

```diff
@@ -31,11 +31,11 @@ import NoLiveData from './NoLiveData'
 const NetworkingCharts = ({ netStats, isRunning, id }) => {
   const haveNetworkStats = !!netStats['current.total']
 
-  const used = (netStats['current.total'] && netStats['current.total'].datum) || 0
+  const used = (netStats['current.total'] && netStats['current.total'].datum && netStats['current.total'].datum[0]) || 0
```

Comment:
@bamsalem Why don't we need to check here if `netStats['current.total'].datum` is an array, too?

Reply:
You're right it should be checked, I added that thanks :)